### PR TITLE
SA-1896: Release Scheduled Reports

### PR DESCRIPTION
**Pull request by kchu1337**
### What Changed

- Removed the flag 'allow_scheduled_reports'

### How To Test
 - Use an account without the feature flag.
 - Check that schedule report is an option in the '...' in the view all reports modal. 
 - Check that you can see the scheduled report modal from the /signals/analytics
 - Check that you can edit a report from the  scheduled report modal.
 - Check that you can create a report from the  scheduled report modal.

### To Do

- [ ] Remove feature flag in confluence doc.

### CODE REVIEW

File: src/pages/reportBuilder/helpers/scheduledReports.js [hunk 1]
```diff
@@ -53,9 +53,11 @@ export const getDefaultValues = (scheduledReport, users) => {
...
   const formattedMinute = formatMinuteString(minute);
   const time = `${formattedHour}:${formattedMinute}`;
-  const fullRecipients = recipients.map(recipient => {
-    return users.find(({ name }) => name === recipient);
-  });
+  const fullRecipients = recipients
+    .map(recipient => {
+      return users.find(({ name }) => name === recipient);
+    })
+    .filter(Boolean);
```
Comment (kchu1337) [inline src/pages/reportBuilder/helpers/scheduledReports.js:60]: Removes users who cannot be found.

**Review by nicklemmon — APPROVED** [review]
@kchu1337 This looks good to me - test failure appears to be a flaky test.

**Review by nicklemmon — CHANGES_REQUESTED** [review]
@kchu1337 I sent you a Slack message about it, but the test failure here is actually something that needs to be addressed before we merge. It's technically a Matchbox problem, but we don't want to introduce test failures if we can help it.

**Review by nicklemmon — APPROVED** [review]
@kchu1337 Looks good to me - thanks for fixing that test.

**Pull request merged**